### ISSUE TITLE
[BUG] Can't delete files in File Manager

### ISSUE THREAD

**Issue by antopolskiy** [innocuous-ai/innocuousbook#2]
Need to be able to remove files to clean up the space.

**Comment by NoamRosenberg** [comment]
Actually I switched this to bug since this is actually designed to work. In AWS this is working fine, but on the Azure platform it's not.

**Comment by schuelu** [comment]
Azure Blob Storage upload and delete files works on az console, throw file manager doesn't works, key headers for file manager authentication resource would be the issue.

![image](https://user-images.githubusercontent.com/37840884/143912719-936b8e28-83ab-48a9-8db4-ffd64d5424f6.png)





![image](https://user-images.githubusercontent.com/37840884/143913051-9aad6cff-04ce-47ae-8c3e-c7365fc6356f.png)






![image](https://user-images.githubusercontent.com/37840884/143914185-61a11874-f376-4033-92fd-a808bfe4472f.png)




![image](https://user-images.githubusercontent.com/37840884/144066742-cab13c96-be45-41df-9b1a-a39445ccf48a.png)

**Comment by NoamRosenberg** [comment]
@antopolskiy we're seeing an issue with only specific kinds of files. Like sometimes pt, json or yaml won't delete. Is the case with you as well?

EDIT: 
An update on this bug. After some testing, It occurs when the last file remaining is not a standard file. The delete is indeed successful but the bug happens when reloading the page. If you click the file afterwards you can see the folder is empty. If you refresh the folder is gone.

**Comment by antopolskiy** [comment]
Yes, so this is embarrassing, but my actual problem was that I didn't realize that there is a context menu if I click right mouse button. I thought that I couldn't delete files at all. So on my side there are no issues right now.

**Comment by NoamRosenberg** [comment]
Thanks @antopolskiy, I will leave this open anyway because of a rare bug that does occur when trying to delete files.
